README.md

```diff
@@ -1,4 +1,10 @@
-# GoldenEye [![JCenter](https://api.bintray.com/packages/infinum/android/goldeneye/images/download.svg)](https://bintray.com/infinum/android/goldeneye/_latestVersion)
+## Deprecation notice
```

Comment:
MAKE THIS A LARGER HEADER WITH UPPERCASE TO INCREASE VISIBILITY

Reply:
I didn't wanna yell, but ok 😄